changelogs/fragments/75142-ssh-retries-collision.yml

```diff
@@ -0,0 +1,7 @@
+bugfixes:
+  - >-
+    task_exector/ssh_connection - use the ``retries`` value from ``ssh_connection`` settings,
```

Comment:
Is this a typo in the change-log entry? `task_exector` => `task_executor` ?

Reply:
Yes, thank you for catching that.